addons/l10n_in/data/account_tcs_tax_template_data.xml

```diff
@@ -0,0 +1,393 @@
+        <field name="active" eval="False"/>
+        <field name="tax_group_id" ref="tcs_group"/>
+        <field name="chart_template_id" ref="indian_chart_template_standard"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),             (0,0, {                 'factor_percent': 100,                 'repartition_type': 'base',             }),              (0,0, {                 'factor_percent': 100,                 'repartition_type': 'tax',                 'account_id': ref('p11245'),                 'plus_report_line_ids': [ref('tcs_report_line_section_206c_1f_mv')],             }),         ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),             (0,0, {                 'factor_percent': 100,                 'repartition_type': 'base',             }),              (0,0, {                 'factor_percent': 100,                 'repartition_type': 'tax',                 'account_id': ref('p11245'),                 'minus_report_line_ids': [ref('tcs_report_line_section_206c_1f_mv')],             }),         ]"/>
+    </record>
+    <record id="tcs_5_us_206c_1g_som" model="account.tax.template">
+        <field name="name">TCS @5% u/s 206C(1G): Sum of money (above 7 lakhs) for remittance out of India</field>
+        <field name="description">TCS @5% u/s 206C(1G): Sum of money (above 7 lakhs) for remittance out of India</field>
+        <field name="type_tax_use">sale</field>
+        <field name="amount_type">percent</field>
+        <field name="amount">5</field>
+        <field name="tax_scope">consu</field>
+        <field name="active" eval="False"/>
+        <field name="tax_group_id" ref="tcs_group"/>
+        <field name="chart_template_id" ref="indian_chart_template_standard"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),             (0,0, {                 'factor_percent': 100,                 'repartition_type': 'base',             }),              (0,0, {                 'factor_percent': 100,                 'repartition_type': 'tax',                 'account_id': ref('p11245'),                 'plus_report_line_ids': [ref('tcs_report_line_section_206c_1g_som')],             }),         ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),             (0,0, {                 'factor_percent': 100,                 'repartition_type': 'base',             }),              (0,0, {                 'factor_percent': 100,                 'repartition_type': 'tax',                 'account_id': ref('p11245'),                 'minus_report_line_ids': [ref('tcs_report_line_section_206c_1g_som')],             }),         ]"/>
+    </record>
+    <record id="tcs_5_us_206c_1g_soaotpp" model="account.tax.template">
+        <field name="name">TCS @5% u/s 206C(1G): Seller of an overseas tour program package</field>
+        <field name="description">TCS @5% u/s 206C(1G): Seller of an overseas tour program package</field>
```

Comment:
if description == name, you can leave out description, but it might be interesting to enter it in the invoice in a shorter format with the many2many_tags on the invoice line